gulpfile.js

```diff
@@ -169,21 +171,21 @@ gulp.task('replace-versionned-assets-in-assets', function () {
     .pipe(collect({
       replaceReved: true,
       dirReplacements: {
-        '/wp-content/themes/visible/dist': config.productionAssetURL
+        fullDistPath : config.productionAssetURL
```

Comment:
This won't work. In javascript this would make the key be `'fullDistPath'`, not the value of the variable. You'd want to do: 

```js
var dirReplacements = {};
dirReplacements[fullDistPath] = config.productionAssetURL;

// ...

.pipe(collect({
  dirReplacements: dirReplacements
```